function gy_exceptions_throw(msg) {
  throw msg;
}
